include/omen/libraries/datastructs/list.h

```diff
@@ -2,6 +2,7 @@
 #define LIST_H
 
 #include <stdbool.h>
+#include <stdio.h>
 
```

Comment:
Remember we cannot use standard libraries (At least I think so @TretornESP  please enlighten us), so include our own stdbool. Also I don't think stdio is necessary here.